File: Python/006.py
def solve(n):
    s1 = n * (n + 1) * (2 * n + 1) // 6
    s2 = (n * (n + 1) // 2) ** 2
    return s2 - s1

# n = 10
n = 100
print(solve(n))

File: C++/006.cpp
#include <iostream>
#include <cmath>
using namespace std;

int solve(int n)
{
    int s1 = n * (n + 1) * (2 * n + 1) / 6;
    int s2 = pow(n * (n + 1) / 2, 2);
    return s2 - s1;
}

int main()
{
    // int n = 10;
    int n = 100;
    cout << solve(n) << endl;
}
